libs/libcommon/src/libcommon/config.py

```diff
@@ -40,72 +40,72 @@
 if TYPE_CHECKING:
     from libcommon.processing_graph import ProcessingGraphSpecification
 
-ASSETS_BASE_URL = "assets"
-ASSETS_STORAGE_DIRECTORY = None
-ASSETS_S3_FOLDER_NAME = "assets"
+ASSETS_BASE_URL = "http://localhost/assets"
+ASSETS_FOLDER_NAME = "assets"
+ASSETS_STORAGE_ROOT = "/storage"
+ASSETS_STORAGE_PROTOCOL = "file"
 
 
 @dataclass(frozen=True)
 class AssetsConfig:
     base_url: str = ASSETS_BASE_URL
-    storage_directory: Optional[str] = ASSETS_STORAGE_DIRECTORY
-    s3_folder_name: str = ASSETS_S3_FOLDER_NAME
+    folder_name: str = ASSETS_FOLDER_NAME
+    storage_protocol: str = ASSETS_STORAGE_PROTOCOL
+    storage_root: str = ASSETS_STORAGE_ROOT
 
     @classmethod
     def from_env(cls) -> "AssetsConfig":
         env = Env(expand_vars=True)
         with env.prefixed("ASSETS_"):
             return cls(
                 base_url=env.str(name="BASE_URL", default=ASSETS_BASE_URL),
-                storage_directory=env.str(name="STORAGE_DIRECTORY", default=ASSETS_STORAGE_DIRECTORY),
-                s3_folder_name=env.str(name="S3_FOLDER_NAME", default=ASSETS_S3_FOLDER_NAME),
+                folder_name=env.str(name="FOLDER_NAME", default=ASSETS_FOLDER_NAME),
+                storage_protocol=env.str(name="STORAGE_PROTOCOL", default=ASSETS_STORAGE_PROTOCOL),
+                storage_root=env.str(name="STORAGE_ROOT", default=ASSETS_STORAGE_ROOT),
             )
 
 
-S3_BUCKET = "hf-datasets-server-statics"
 S3_ACCESS_KEY_ID = None
 S3_SECRET_ACCESS_KEY = None
-S3_REGION = "us-east-1"
```

Comment:
the region is not needed anymore? what occurs if we use a bucket in another region?

Reply:
I think it is not needed when using fsspec. See https://github.com/fsspec/s3fs/blob/2c074502c2d6a9be0d3f05eb678f4cc5add2e7e5/s3fs/core.py#L171 params.
Anyway, I will add the region as part of client_kwargs to prevent issues.